x-pack/plugins/security_solution/public/resolver/view/graph_controls.tsx

```diff
@@ -125,12 +125,13 @@ const GraphControlsComponent = React.memo(
         className={className}
         graphControlsBackground={colorMap.graphControlsBackground}
         graphControlsIconColor={colorMap.graphControls}
+        data-test-subj="resolver:graph-controls"
       >
         <EuiPanel className="panning-controls" paddingSize="none" hasShadow>
           <div className="panning-controls-top">
             <button
               className="north-button"
-              data-test-subj="north-button"
+              data-test-subj="resolver:graph-controls:north-button"
               title="North"
```

Comment:
:reminder_ribbon: later to fix these to better things like "Pan the scene up" or something. North is not a good description for what this does.

Reply:
Yea, we can talk about this during office hours :)